CondTools/L1Trigger/interface/L1ConfigOnlineProdBase.h

```diff
@@ -171,7 +176,7 @@ bool L1ConfigOnlineProdBase<TRcd, TData>::getObjectKey(const TRcd& record, std::
   // already in ORCON.
   edm::ESHandle<L1TriggerKey> key;
   try {
-    keyRcd.get(key);
+    key = iRecord.template getHandle(l1TriggerKeyToken_);
```

Comment:
what is `template` doing here? I'm confused a bit, sorry, shouldnt this be
`key = iRecord.getHandle(l1TriggerKeyToken_);`

(and also I think `key` could be made `const`)

Reply:
Thanks for spotting this. My mistake! fixed it.